src/Components/Postulants/Form/index.js

```diff
@@ -0,0 +1,714 @@
+  const [emailValue, setEmailValue] = useState('');
+  const [passwordValue, setPasswordValue] = useState('');
+  const [contactFromValue, setContactFromValue] = useState('');
+  const [contactToValue, setContactToValue] = useState('');
+  const [adressValue, setAdressValue] = useState('');
+  const [birthdayValue, setBirthdayValue] = useState('');
+  const [availableValue, setAvailableValue] = useState('');
+  const [phoneValue, setPhoneValue] = useState('');
+  const [primarySDValue, setPrimarySDValue] = useState('');
+  const [primaryEDValue, setPrimaryEDValue] = useState('');
+  const [primarySchoolValue, setPrimarySchoolValue] = useState('');
+  const [secondarySDValue, setSecondarySDValue] = useState('');
+  const [secondaryEDValue, setSecondaryEDValue] = useState('');
+  const [secondarySchoolValue, setSecondarySchoolValue] = useState('');
+  const [tertiarySDValue, setTertiarySDValue] = useState('');
+  const [tertiaryEDValue, setTertiaryEDValue] = useState('');
+  const [tertiaryDescriptionValue, setTertiaryDescriptionValue] = useState('');
+  const [tertiaryInstituteValue, setTertiaryInstituteValue] = useState('');
+  const [universitySDValue, setUniversitySDValue] = useState('');
+  const [universityEDValue, setUniversityEDValue] = useState('');
+  const [universityDescriptionValue, setUniversityDescriptionValue] = useState('');
+  const [universityInstituteValue, setUniversityInsituteValue] = useState('');
+  const [informalSDValue, setInformalSDValue] = useState('');
+  const [informalEDValue, setInformalEDValue] = useState('');
+  const [informalDescriptionValue, setInformalDescriptionValue] = useState('');
+  const [informalInstituteValue, setInformalInstituteValue] = useState('');
+  const [workExperienceCompanyValue, setWorkExperienceCompanyValue] = useState('');
+  const [workExperienceSDValue, setWorkExperienceSDValue] = useState('');
+  const [workExperienceEDValue, setWorkExperienceEDValue] = useState('');
+  const [workExperienceDescriptionValue, setWorkExperienceDescriptionValue] = useState('');
+
+  if (postulantId) {
+    useEffect(() => {
+      fetch(`${process.env.REACT_APP_API}/postulants?_id=${postulantId}`)
+        .then((response) => response.json())
+        .then((response) => {
+          autoFill(response);
+        });
+    }, []);
+  }
+
+  const autoFill = (data) => {
+    const fillData = data.data[0];
+    const fillPrimStudy = data.data[0].studies.primaryStudies;
+    const fillSecStudy = data.data[0].studies.secondaryStudies;
+    const fillTerStudy = data.data[0].studies.tertiaryStudies[0];
+    const fillUniStudies = data.data[0].studies.universityStudies[0];
+    const fillInfStudies = data.data[0].studies.informalStudies[0];
+    const fillWorkExp = data.data[0].workExperience[0];
+
+    let from = fillData.contactRange?.from.toString();
+    let to = fillData.contactRange?.to.toString();
+    from = `${from.slice(0, 2)}:${from.slice(2)}`;
+    to = `${to.slice(0, 2)}:${to.slice(2)}`;
```

Comment:
update the Schema to make it easy, just use the format that the time input return

Reply:
added RegExp and now hours are stored as 'string' in mongoose